string img_path
string cam_type
---
bool success
string message

